### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'country_code') (2171)

### ISSUE THREAD

**Issue by DylVadim** [woocommerce/woocommerce-paypal-payments#1205]
### Describe the Bug

Send alert in Checkout:
Something went wrong. Please try again or choose another payment source. 
Error in Console:
TypeError: Cannot read properties of undefined (reading 'country_code') 

### To Reproduce

1. Go to Checkout
2. Click on button "PayPal"
3. See error

### Screenshots

![image](https://user-images.githubusercontent.com/54818214/220580939-4d0177eb-908f-4f42-967b-d4e49bf0cc56.png)

### Expected Behavior

Redirect to the PayPal platform for making a payment

### Actual Behavior

Send alert in Checkout:
Something went wrong. Please try again or choose another payment source. 
Error in Console:
TypeError: Cannot read properties of undefined (reading 'country_code') 

### Environment

 - WordPress Version 6.1.1
 - WooCommerce Version 7.4.0
 - Plugin Version 2.0.2
 - Browser Chrome 110.0.5481.104 x64
 - Theme Woodmart 7.1.3

### Additional Details

<details><summary>System status</summary>

  ```
  TypeError: Cannot read properties of undefined (reading 'country_code') 
  ```

</details>

**Comment by DylVadim** [comment]
In the WooCommerce settings, you need to set the following parameters:
![image](https://user-images.githubusercontent.com/54818214/220589613-ae8e50d7-5f0a-4643-b206-7c9a02950639.png)

**Comment by DylVadim** [comment]
for some reason, the above described problem persists for some users:
![332133243_734756291519664_6695303533807517111_n (2)](https://user-images.githubusercontent.com/54818214/224767844-1f0bc8e9-efd1-464c-a58d-4bba6df4878c.jpg)
![332133243_734756291519664_6695303533807517111_n](https://user-images.githubusercontent.com/54818214/224767859-82e6646b-66e3-42f9-a70c-a7249d3c2242.jpg)
![333739700_1918370338508930_919817929481323996_n (2)](https://user-images.githubusercontent.com/54818214/224767863-64a96d75-2009-4292-917a-319b3cdbc00f.jpg)
![333739700_1918370338508930_919817929481323996_n](https://user-images.githubusercontent.com/54818214/224767865-9460e9bd-b8b7-4dea-8083-40a940386bad.jpg)

**Comment by kmligue** [comment]
im having similar error. were you able to fix it?

**Comment by DylVadim** [comment]
No, i could not fix the error, the client decided to switch back to the Shopify platform.
